torch/csrc/jit/passes/quantization.h

```diff
@@ -118,6 +118,14 @@ TORCH_API void FoldConvBatchNorm2d(const script::Module& module);
  */
 TORCH_API void FoldQuantizeCallIntoBuffer(script::Module& module, const std::string& method_name);
 
+/** \brief Insert pack and unpack function in all graphs
+ *   of module
+ */
+TORCH_API void InsertPackUnpack(script::Module& module);
+
+/** \brief Insert pack and unpack function in graph
+ */
+TORCH_API void InsertPackUnpack(std::shared_ptr<Graph>& graph);
```

Comment:
This doesn't need to be an external API.

Reply:
hmm, I want this for easier testing, since this is just a graph transformation

Reply:
That's reasonable, just update the comment then.